my story.txt

```diff
@@ -30,3 +30,4 @@ its adhg
 no, its xww
```

Comment:
this can be removed!

Reply:
sure. just the 2 of us...we can make it if we try.